doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -196,6 +206,45 @@ def get_model_source(url):
         return "Source"
 
 
+def create_proprietary_model_table():
```

Comment:
this is wild, I would just use f strings with triple quotes so that you can have line breaks and have a better visual of what the final string output would look like

Reply:
thanks, I wasn't refactoring the `doc_utils` file too much and followed existing patterns, but yes updated this section for sure

Reply:
can you run this by @judyheflin , as this is a documentation change?